src/expandable-block/block.scss

```diff
@@ -71,34 +70,45 @@
       }
     }
 
-    &:focus {
-      outline: 0;
-
-      @include themed {
-        box-shadow: t(iui-focus-box-shadow);
-      }
-    }
+    @include iui-focus;
   }
 
   &.iui-with-caption > .iui-header > .iui-icon {
     margin-top: $iui-baseline;
   }
 
   .iui-expandable-content {
-    transform: scaleY(0);
-    transform-origin: top;
     overflow: hidden;
-    padding: 0;
-    opacity: 0;
-    height: 0;
-    @media (prefers-reduced-motion: no-preference) {
-      transition: transform $iui-speed-fast ease-in-out;
-    }
+    box-sizing: border-box;
+
     @include themed {
       border-bottom: 1px solid t(iui-color-background-4);
       border-right: 1px solid t(iui-color-background-4);
       border-left: 1px solid t(iui-color-background-4);
     }
+
+    > div {
+      padding: $iui-baseline $iui-sm;
+    }
+
+    // #region react-transition-group classes
```

Comment:
When you wrote it as a region, now it really makes sense to have a `mixin` and reuse it in `Table`.

Reply:
Done